CromIAM/src/main/scala/cromiam/webservice/WomtoolRouteSupport.scala

```diff
@@ -2,20 +2,18 @@ package cromiam.webservice
 
 import akka.http.scaladsl.server.Directives._
 import cromiam.cromwell.CromwellClient
-import cromwell.api.model._
+import cromiam.sam.SamClient
 
 trait WomtoolRouteSupport extends RequestSupport {
   // When this trait is mixed into `CromIamApiService` the value of `cromwellClient` is the reader (non-abort) address
   val cromwellClient: CromwellClient
+  val samClient: SamClient
 
   val womtoolRoutes =
     path("api" / "womtool" / Segment / "describe") { _ =>
       post {
-        extractStrictRequest { req =>
```

Comment:
The new code path no longer seems to mention "strict"... is that ok?

Reply:
Ah, the `extractUserAndRequest` function [quietly makes use of `extractStrictRequest` under the covers](https://github.com/broadinstitute/cromwell/blob/develop/CromIAM/src/main/scala/cromiam/webservice/RequestSupport.scala#L29-L35). I will rename it for clarity.